### PR TITLE
86due2wwb

### PR DESCRIPTION
**Pull request by lualreye**
- Create a new route in router
- Add dashboard route in navbar
- Fix navbar styles

### CODE REVIEW

File: src/views/app/Dashboard/Index.vue
Comment (yeyodev1) [inline src/views/app/Dashboard/Index.vue:1]: only after tomorrow we have to have the correct view

File: src/router/index.ts [hunk 1]
```diff
@@ -87,6 +88,14 @@ const routes: Array<RouteRecordRaw> = [
     name: 'App',
     component: AppLayout,
     children: [
+      {
```
Comment (yeyodev1) [inline src/router/index.ts:91]: awesome

File: src/views/app/Dashboard/Index.vue
Comment (lualreye) [reply src/views/app/Dashboard/Index.vue:1]: of course

File: src/layouts/AppLayout.vue [hunk 2]
```diff
@@ -30,9 +49,47 @@ const pageTitle = computed(() => route.meta.title);
...
+          position: absolute;
+          top: 100%;
+          right: 0;
+          margin-top: 8px;
+          :deep(.crush-button.crush-primary) {
+            background-color: $white;
+            border: 1px solid $grey;
+          }
+
+          span {
```
Comment (yeyodev1) [inline src/layouts/AppLayout.vue:84]: It is recommended that you use class in this tag cause our vue version 3.4.21 has a conflict with styles and is possible that you are changing things that you are not considering

File: src/layouts/AppLayout.vue [hunk 3]
```diff
@@ -2,20 +2,39 @@
...
     <div class="layout-container-content">
-      <h1 class="title">
-        {{ pageTitle }}
-      </h1>
+      <div class="header">
+        <h1 class="title">
+          {{ pageTitle }}
+        </h1>
+        <div class="profile-container">
+          <i class="fa-solid fa-circle-user profile-icon"></i>
```
Comment (yeyodev1) [inline src/layouts/AppLayout.vue:28]: This should close like this '/>' unless you're wrapping something, are you wrapping something?

**Review by yeyodev1 — CHANGES_REQUESTED** [review]
check my changes

**Pull request merged**